ui/src/containers/VisualiseResults/SourceResults.js

```diff
@@ -141,50 +209,75 @@ const SourceResult = ({
     return <NoData />;
   }
 
+  const showStatsAtTop = visualisation.get('showStatsAtTop', true);
+  const showStatsAtBottom = visualisation.get('showStatsAtBottom', true);
+
+  const stats = calcStats(formattedResults);
+
+  // TODO: Show a message telling that the result might be limited
```

Comment:
Can we do this?

Reply:
@Ian247 @ryansmith94 When the size of results from API server is 10,000, the result may be limited. That means that `total`, `avg`, `min` and `row count` are probably not accurate numbers. Should we display a message if the result size is 10,000? If yes, can you specify the message text?

Reply:
Yeah I think we should display a message, something like "Totals calculated from the first 10,000 records".

Reply:
Are we saying that we could only total up 10,000 statements?

Reply:
No just 10,000 rows.